data_validation/clients.py

```diff
@@ -27,6 +27,7 @@
 from ibis.backends.mysql.client import MySQLClient
 from ibis.backends.pandas.client import PandasClient
 from ibis.backends.postgres.client import PostgreSQLClient
+import third_party.ibis.ibis_postgres.client
```

Comment:
nit: Move the import statement up to line 25 to keep all the full imports together